### ISSUE TITLE
User password accepts and uses control characters

### ISSUE THREAD

**Issue by tlewand42** [CeON/fairchive#2262]
It is currently possible to include control characters (UNICODE 0x01 to 0x31, NUL 0x00 character is excluded) in passwords.
Passwords containing control characters are accepted without further changes.
It is also possible to change password to new one different from the previous one by use od control characters only. The change is accepted as valid change.
Password pasted with control characters behaves differently than the same password pasted without control characters.

tl;dr: control characters behave just as any other character when they are a part of a user password.
